AK/String.cpp

```diff
@@ -113,8 +113,9 @@ Vector<String> String::split_limit(char separator, size_t limit, SplitBehavior s
         char ch = characters()[i];
         if (ch == separator) {
             size_t sublen = i - substart;
+            bool keep_separator = has_flag(split_behavior, SplitBehavior::KeepSeparator);
```

Comment:
This could be moved before the loop, so it doesn't get calculated repeatedly.

Reply:
Nice catchhh.